### DISCUSSION TITLE
Miller at csv,conf,v7

### DISCUSSION THREAD

**Discussion by johnkerl** [johnkerl/miller#1268] [General]
* Schedule/abstract: https://csvconf.com/schedule/#john-kerl
* Slides [johnkerl.org/miller,csv,conf,v7/miller,csv,conf,v7.pdf](https://johnkerl.org/miller,csv,conf,v7/miller,csv,conf,v7.pdf)

I'll post any Q&A follow-ups here

**Comment by aborruso** [comment]
It would be a dream to be at this conference. Good luck.
You will amaze them all, because Miller is a jewel, still little known compared to what it deserves
Going to this conference will make it know more.

Well done, John

**Reply by trantor** [reply]
Wow, spectacular. Can't wait to see the material @johnkerl !

**Reply by johnkerl** [reply]
@trantor slides: [johnkerl.org/miller,csv,conf,v7/miller,csv,conf,v7.pdf](https://johnkerl.org/miller,csv,conf,v7/miller,csv,conf,v7.pdf)

**Reply by aborruso** [reply]
Is it only for @trantor? 🤣

I can't wait to see the video of this presentation as well.
Giuseppe Sollazzo told me that it was very beautiful

**Comment by johnkerl** [comment]
https://youtu.be/G0_rFYz6rJg

**Comment by johnkerl** [comment]
Unfortunately the audio is missing for some of the middle.

But, as chance would have it I did a dry-run/prep recording before the conference.

If you start the conference video about 7:50
https://www.youtube.com/watch?v=G0_rFYz6rJg
and mute it, then this about 6:40
https://drive.google.com/file/d/1DqvR7a6oMJRA34LX9XRqWJ8x_ON6XqRv/view?usp=drive_link
and let this run in parallel, things will more or less line up.

**Reply by aborruso** [reply]
> and mute it, then this about 6:40
> [drive.google.com/file/d/1DqvR7a6oMJRA34LX9XRqWJ8x_ON6XqRv/view?usp=drive_link](https://drive.google.com/file/d/1DqvR7a6oMJRA34LX9XRqWJ8x_ON6XqRv/view?usp=drive_link)
> and let this run in parallel, things will more or less line up.

I have for it

![image](https://github.com/johnkerl/miller/assets/30607/bccfa7c5-3506-4d4e-94e9-c74afbbc26e3)

**Reply by johnkerl** [reply]
@aborruso let's try https://drive.google.com/file/d/1DqvR7a6oMJRA34LX9XRqWJ8x_ON6XqRv/view?usp=sharing

**Reply by aborruso** [reply]
> @aborruso let's try [drive.google.com/file/d/1DqvR7a6oMJRA34LX9XRqWJ8x_ON6XqRv/view?usp=sharing](https://drive.google.com/file/d/1DqvR7a6oMJRA34LX9XRqWJ8x_ON6XqRv/view?usp=sharing)

It works, thank you
